vm/src/stdlib/os.rs

```diff
@@ -1228,6 +1228,22 @@ mod _os {
         Ok(())
     }
 
+    #[cfg(unix)]
+    #[pyfunction]
+    fn getloadavg(vm: &VirtualMachine) -> PyResult<(f64, f64, f64)> {
+        let mut loadavg = [0f64; 3];
+
+        // Safety: loadavg is on stack and only write by `getloadavg` and are freed
+        // after this function ends.
+        unsafe {
+            if libc::getloadavg(&mut loadavg[0] as *mut f64, 3) != 3 {
+                return Err(vm.new_os_error("Load averages are unobtainable".to_string()));
+            }
+        }
+
+        return Ok((loadavg[0], loadavg[1], loadavg[2]));
```

Comment:
Oh, and I think this is what's failing the linter, you don't need the `return` here - you can just leave off the last semicolon and it returns the value from the function.

Reply:
Yes, I will setup clippy in my develop environment to avoid this kind error from happening again.